Source/WebCore/bindings/js/WindowProxy.h

```diff
@@ -91,11 +94,13 @@ class WindowProxy : public RefCounted<WindowProxy> {
 
 private:
     explicit WindowProxy(AbstractFrame&);
+    explicit WindowProxy(RefPtr<RemoteDOMWindow>);
 
     JSWindowProxy& createJSWindowProxy(DOMWrapperWorld&);
     WEBCORE_EXPORT JSWindowProxy& createJSWindowProxyWithInitializedScript(DOMWrapperWorld&);
 
     WeakPtr<AbstractFrame> m_frame;
+    RefPtr<RemoteDOMWindow> m_remoteWindow;
```

Comment:
This doesn't look right. We should be getting the remote window from m_frame (which would be a RemoteFrame).